tests/e2e/tests/test_redirects.py

```diff
@@ -168,34 +81,29 @@ def test_that_checks_redirect_using_incorrect_query_values(self, base_url):
         assert urlencode(param) == parsed_url.query, \
             self.response_info_failure_message(base_url, param, response)
 
-    @pytest.mark.parametrize('os', OS)
-    @pytest.mark.parametrize('lang', LOCALES)
-    def test_that_checks_redirect_using_locales_and_os(
-        self,
-        base_url,
-        lang,
-        os
-    ):
-        # Ja locale has a special code for mac
+    @pytest.mark.parametrize('os', _os)
+    @pytest.mark.parametrize('locale', _locales)
+    def test_verify_locales_redirect_to_the_expected_product(self, base_url, locale, os):
+        """Verifies the downloaded version of Firefox matches the expected version number
+        and filename when Firefox is requested for a specific OS and locale.
+
+        The test verifies the following aliases: firefox-latest, firefox-esr-latest,
+        firefox-nightly-latest, firefox-beta-latest, firefox-beta-latest, firefox-aurora-latest.
+        """
+        lang = locale.lang
+        # Ja locale has a OSX specific locale
         if lang == 'ja' and os == 'osx':
             lang = 'ja-JP-mac'
```

Comment:
I hate this mozillizm. 🤣 Nothing we can do about it. I see this patter EVERYWHERE. :)